src/rootPages/Designer/properties/process/ABProcessTaskServiceCalculate.js

```diff
@@ -0,0 +1,236 @@
+import UI_Class from "../../ui_class";
+
+export default function (AB) {
+  const UIClass = UI_Class(AB);
+  var L = UIClass.L();
+
+  class UIProcessServiceCalculate extends UIClass {
+    constructor() {
+      super("properties_process_service_calculate", {
+        name: "",
+        formulaText: "",
+        variableList: "",
+
+        variablePopup: "",
+        operatorPopup: "",
+      });
+
+      this.element = null;
+    }
+
+    static get key() {
+      return "Calculate";
+    }
+
+    ui() {
+      const ids = this.ids;
+
+      webix.ui({
+        id: ids.variablePopup,
+        view: "popup",
+        hidden: true,
+        body: {
+          id: ids.variableList,
+          view: "list",
+          template: (item) => {
+            return item.value;
+          },
+          data: "",
+          on: {
+            onItemClick: function (id) {
+              var component = this.getItem(id);
+
+              insertFormula(`{${component.value}}`);
+
+              $$(ids.variablePopup).hide();
+            },
+          },
+        },
+      });
+
+      let insertFormula = (message) => {
+        let formula = $$(ids.formulaText).getValue();
+
+        $$(ids.formulaText).setValue(`${formula}${message} `);
+      };
+
+      webix.ui({
+        id: ids.operatorPopup,
+        view: "popup",
+        hidden: true,
+        width: 180,
+        body: {
+          view: "list",
+          template: (item) => {
+            var template = "";
+
+            if (item.icon) {
+              template += `<i class="fa fa-${item.icon}" aria-hidden="true"></i> `;
+            }
+
+            if (item.label) {
+              template += item.label;
+            }
+
+            return template;
+          },
+          data: [
+            {
+              label: L("+ Adds"),
+              symbol: "+",
+            },
+            {
+              label: L("- Subtracts"),
+              symbol: "-",
+            },
+            {
+              label: L("* Multiples"),
+              symbol: "*",
+            },
+            {
+              label: L("/ Divides"),
+              symbol: "/",
+            },
+            {
+              label: L("( Open Bracket"),
+              symbol: "(",
+            },
+            {
+              label: L(") Closed Bracket"),
+              symbol: ")",
+            },
+          ],
+          on: {
+            onItemClick: function (id, e, node) {
+              var component = this.getItem(id);
+
+              insertFormula(component.symbol);
+
+              $$(ids.operatorPopup).hide();
+            },
+          },
+        },
+      });
+      let labelWidth = 120;
+      return {
+        rows: [
+          {
+            //     id: id,
+            view: "form",
+            elementsConfig: {
+              labelWidth: labelWidth,
+            },
+            elements: [
+              {
+                id: ids.name,
+                view: "text",
+                label: L("*Name"),
```

Comment:
Please remove the *'s in labels. We used this pattern in v1, but in v2 we don't want that. Should be L("Formula")